src/friktion/ids.ts

```diff
@@ -3,3 +3,10 @@ import { PublicKey } from "@solana/web3.js";
 export const VOLT_PROGRAM_ID = new PublicKey("VoLT1mJz1sbnxwq5Fv2SXjdVDgPXrb9tJyC8WpMDkSp");
 
 export const VOLT_FEE_OWNER = new PublicKey("3KjJiWBfaw96qGhysq6Fc9FTxdPgPTNY6shM7Bwfp8EJ");
+
+export const IS_STABLE = new Map<string, boolean>([
+  ["EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v", true],
```

Comment:
1. What are these account?
2. Is the boolean necessary?

Reply:
<img width="454" alt="Screen Shot 2022-12-06 at 6 04 53 PM" src="https://user-images.githubusercontent.com/35906798/205881062-b5b44814-4010-450c-8ee7-75d4e9772efa.png">
The keys is stable coins' mint address